src/Raven.Server/Documents/Indexes/Persistence/Corax/CoraxIndexPersistence.cs

```diff
@@ -119,19 +119,24 @@ public override void Clean(IndexCleanup mode)
     public override void Initialize(StorageEnvironment environment)
     {
     }
-
-    public override void OnInitializeComplete()
+    
+    public override void OnBeforeExecuteIndexing(IndexingStatsAggregator indexingStatsAggregator)
+    {
+        using (indexingStatsAggregator.CreateScope().For(IndexingOperation.Corax.DictionaryTraining))
```

Comment:
Does it require any work in the studio?

Reply:
Yes, there is a flag `Studio Required` in the ticket and requirements in the PR description.